src/embind/emval.js

```diff
@@ -530,6 +530,7 @@ var LibraryEmVal = {
 
 #if ASYNCIFY
   _emval_await__deps: ['$Emval', '$Asyncify'],
+  _emval_await__sig: 'pp',
```

Comment:
The idea is that these signatures should be auto-generated by `tools/gen_sig_info.py`..

Reply:
Looks like maybe adding `'ASYNCIFY': 1` to line 380 might do the trick